docker-compose.local.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.3'
+services:
+    web:
+        build: .
+        image: geokrety:local
+        ports:
+            - "8000:80"
+        volumes:
+            - ${PWD}/website/:/var/www/html/:rw
+            - ${PWD}/configs/konfig-mysql.docker.php:/var/www/html/templates/konfig-mysql.php:ro
+            - ${PWD}/configs/konfig-local.tmpl.php:/var/www/html/templates/konfig-local.php:ro
+            - ${PWD}/configs/ssmtp.tmpl.conf:/etc/ssmtp/ssmtp.conf:ro
```

Comment:
This file probably need some tuning before being used (I'm thinking at least about `mailhub=mail` parameter).

Reply:
Works for me as is. Maybe emails don't work, this can be true. But I wanted to just check main website logic, not emails =)